client/components/cart/Cart.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import {connect} from 'react-redux';
-import {fetchCart} from '../store/redux/cart';
+import {fetchCart} from '../../store/redux/cart/cart';
 
 class Cart extends React.Component {
   constructor() {
```

Comment:
add documentation to line 22 letting the group know that the functions are working with your db where you forced in associations.